packages/xml-views-validation/src/validators/document/non-unique-id.ts

```diff
@@ -91,15 +97,13 @@ class IdsCollectorVisitor implements XMLAstVisitor {
 // We only care about UI5 elements/controls IDs when check non-unique IDs
 // `id` attributes in these: **known** namespaces which are sometimes used
 // in UI5 xml-views are definitively not relevant for this validation
-const whiteListedNamespaces: Record<string, boolean> = {
-  "http://www.w3.org/1999/xhtml": true,
-  "http://www.w3.org/2000/svg": true,
-  "http://schemas.sap.com/sapui5/extension/sap.ui.core.template/1": true,
-};
+const whiteListedNamespaces: Record<string, boolean> = {};
```

Comment:
You can initialize it the same way as before (with the constants inside `[]`)
```js
const whiteListedNamespaces: Record<string, boolean> = {
   [SVG_NS]: true,
   ...
};
```

Reply:
oh nice.